CHANGELOG.md

```diff
@@ -1,8 +1,19 @@
-# dbt_jira VERSION.VERSION
+# dbt_jira v0.16.0
+[PR #122](https://github.com/fivetran/dbt_jira/pull/122) contains the following updates:
+
+## 🚨 Breaking Changes: Bug Fixes 🚨
+- The following fields in the below mentioned models have been converted to a string datatype (previously integer) to ensure classic Jira projects may link issues to epics. In classic Jira projects the epic reference is in a hyperlink form (ie. "https://ulr-here/epic-key") as opposed to an ID. As such, a string datatype is needed to successfully link issues to epics. If you are referencing these fields downstream, be sure to make any changes to account for the new datatype.
+  - `revised_parent_issue_id` field within the `int_jira__issue_type_parents` model
+  - `parent_issue_id` field within the `jira__issue_enhanced` model
```

Comment:
I don't see `parent_issue_id` in the jira__issue_enhanced. Did you mean `issue_id`? Or did you mean to mention a different model, `int_jira__issue_users`?

Also, if you're mentioning int_jira__issue_type_parents, should you mention the changes in `int_jira__issue_epic` and `int_jira__issue_users` too?

Reply:
> I don't see parent_issue_id in the jira__issue_enhanced. Did you mean issue_id? Or did you mean to mention a different model, int_jira__issue_users?

![image](https://github.com/fivetran/dbt_jira/assets/74217849/302e7ade-c48d-45f9-ad3b-8186c5315bdd)


> Also, if you're mentioning int_jira__issue_type_parents, should you mention the changes in int_jira__issue_epic and int_jira__issue_users too?

These are both ephemeral models which which will not result in any destination changes on the customers side. This is why they were not listed.

Reply:
Got it, thanks for clarifying, looks good!